### ISSUE TITLE
How to get loss,acc,val_loss,val_acc on Custom Model Training

### ISSUE THREAD

**Issue by tanzikun** [OlafenwaMoses/ImageAI#409]
model_trainer = ModelTraining()
model_trainer.setModelTypeAsDenseNet()
model_trainer.setDataDirectory(args.dataDirectory)
model_trainer.trainModel(num_objects=2, num_experiments=10, enhance_data=True, batch_size=10, show_network_summary=False)

trainModel havn`t return parameter,I want to get loss,acc,val_loss,val_acc.

thanks
